src/lib.rs

```diff
@@ -33,9 +33,8 @@
 //! the remote command directly. Usually, these are the same, though not always, as highlighted in
 //! the documetantation the individual methods. See also the section below on Remote Shells.
 //!
-//! And finally, our commands never default to inheriting stdin/stdout/stderr, since we expect you
-//! are using this to automate things. Instead, unless otherwise noted, all I/O ports default to
-//! [`Stdio::null`](std::process::Stdio::null).
+//! And finally, our commands is default to inheriting stdin/stdout/stderr to be compatible with
```

Comment:
I don't think this is actually accurate? I think we can just remove this paragraph entirely, since this is now not a difference from `std`.

Reply:
Done, I have removed this paragraph.